bin/yaml/dart.yaml

```diff
@@ -9,6 +9,9 @@ compilers:
       - find -executable -print0 | xargs -0 chmod og+x
     url: https://storage.googleapis.com/dart-archive/channels/stable/release/{{name}}/sdk/dartsdk-linux-x64-release.zip
     targets:
+      - 3.7.3
+      - 3.6.2
+      - 3.5.4
```

Comment:
I replaced this one w/ 3.5.4...is that a problem?

Also: would be REALLY nice if CI caught this! 😄

Reply:
Replaced which one? Sorry no following here. What CI check could catch this do you think?

Reply:
Oh; I see - you removed dart 3.5.3! I'll put it back!

Reply:
(or I can alias dart 3.5.3 to 3.5.4? Our general principle is _never replace compilers_ as it will break short links, but dart might be a language where we don't care about that?) I'll make an alias.

Reply:
I doubt the compiler changed between these two dot releases, but maybe.

I'm a big fan of not breaking links. Thanks! 🙏

Reply:
(to give context: we often see Compiler Explorer used in filing bugs in compilers. Point releases fix bugs, typically, so if we did this (kept latest and aliased instead) for e.g. GCC or clang we'd be gaslighting people who are looking to see if their bug got fixed in gcc 15.2.1 vs 15.2.0 :)